platform/linux-generic/odp_queue_scalable.c

```diff
@@ -1014,6 +1014,77 @@ static void queue_print(odp_queue_t handle)
 	UNLOCK(&queue->s.lock);
 }
 
+#define COL_WIDTH 24
+
+static void queue_print_all(void)
+{
+	uint32_t i, index;
+	const char *name;
+	int status;
+	odp_queue_type_t type;
+	odp_nonblocking_t blocking;
+	odp_queue_op_mode_t enq_mode;
+	odp_queue_op_mode_t deq_mode;
+	odp_queue_order_t order;
+	const char *bl_str;
+	char type_c, enq_c, deq_c, order_c, sync_c;
+	int prio = 0;
+	odp_schedule_sync_t sync = ODP_SCHED_SYNC_PARALLEL;
+
+	ODP_PRINT("\nList of all queues:\n");
+	ODP_PRINT("------------------\n");
+	ODP_PRINT(" idx %-*s type block enq deq ord sync prio\n", COL_WIDTH, "name");
+
+	for (i = 0; i < CONFIG_MAX_QUEUES; i++) {
+		queue_entry_t *queue = &queue_tbl->queue[i];
+
+		if (queue->s.status != QUEUE_STATUS_READY)
+			continue;
+
+		LOCK(&queue->s.lock);
+
+		status   = queue->s.status;
+		index    = queue->s.index;
+		name     = queue->s.name;
+		type     = queue->s.type;
+		blocking = queue->s.param.nonblocking;
+		enq_mode = queue->s.param.enq_mode;
+		deq_mode = queue->s.param.deq_mode;
+		order    = queue->s.param.order;
+
+		UNLOCK(&queue->s.lock);
+
+		if (status < QUEUE_STATUS_READY)
```

Comment:
status != QUEUE_STATUS_READY, right?

Reply:
True, scalable has only one status value for active queues. Fixed in v2.